README.rst

```diff
@@ -175,6 +175,14 @@ For documentation for this project - you're reading it. ;-)
 
 For in-editor documentation aimed at the user, this is in the help.html file.
 
+Metrics
```

Comment:
Hey @kavitakapoor, do you have any preference to wording in the README?
This PR and README are on the **public** repository